__tests__/routes/profile.test.js

```diff
@@ -70,7 +70,7 @@ describe('Profile', () => {
     done();
   });
 
-  test('should not create profile with --taken email and username', async done => {
+  test('should not create profile with --taken email and username', async (done) => {
```

Comment:
Why is that single parameter in the parenthesises?

Reply:
Are changing some rules?

Reply:
Not man I am not

Reply:
<img width="990" alt="screen shot 2019-03-08 at 09 00 13" src="https://user-images.githubusercontent.com/37615728/54013565-4161e780-4182-11e9-929d-27dae05aaa9d.png">